changes/ee/feat-13199.en.md

```diff
@@ -0,0 +1,23 @@
+Implemented the Message Enrichment and Transformation feature.
+
+This allows users to transform incoming messages without the need to define SQL rules in the Rule Engine, just by using simple variform syntax.
+
+For example: if we want to take an incoming message encoded as Avro, decode it to JSON and then take the `tenant` client attribute from the publishing client and prefix it to the topic, before processing the result in Rule Engine, one would only need a transformation with the following configuration:
+
+```hocon
+message_transformation {
+  transformations = [
+    {
+      name = mytransformation
+      enable = true
```

Comment:
When enable is true by default, don’t provide it in examples

Reply:
I'll fix in a follow up.